cmd/shortener/config.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"os"
+)
+
+type configuration struct {
+	flagRunAddr     string
+	flagBaseURLAddr string
+}
+
+var config = &configuration{}
+
+func parseFlags() {
+	flag.StringVar(&config.flagRunAddr, "a", "localhost:8080", "address and port to run server")
+	flag.StringVar(&config.flagBaseURLAddr, "b", "http://localhost:8080", "address and port to run server")
```

Comment:
It would be better if the default value is `127.0.0.0:8080`

Reply:
ok